README.md

```diff
@@ -1,43 +1,41 @@
 # YouTube Playlister
-Program provides information about videos/songs in a YouTube playlist. Idea behind this program is that some videos are sometimes deleted by uploaders, so this is a way to make sure you have saved at least song names (before they are deleted).
+Program provides information about videos/songs in a YouTube playlist. Idea behind this program is that some videos are sometimes deleted by uploaders, so this is a way to make sure you have saved at least song names before they are deleted.
 
 ## Starting the program
 
-#### Console
-If you don't want to clone this project, navigate to `jar` folder (or just  click [here](https://github.com/ajvazz/YouTube-Playlister/blob/master/YT_Playlister/jar/YT_Playlister.jar)) and download .jar file. Then, go to its download location, open terminal/console, and type:  
-`java -cp YT_Playlister.jar App`. Visit _Usage_ to see how to use the program.
+#### 1. Console
+If you don't want to clone this project, navigate to `target` folder (or just  click [here](https://github.com/ajvazz/YouTube-Playlister/blob/dragan/YT_Playlister/target/YouTubePlaylister.jar)) and download this .jar file. Then, go to its download location, open terminal/console, and type:  
+`java -jar YouTubePlaylister.jar playlist_link path_to_output_file`.
 
-#### IntelliJ IDEA
-First, clone this repository with `git clone https://github.com/ajvazz/YouTube-Playlister.git`. Then, open IntelliJ IDEA and import this program (as Maven).  
+**For example**, if you'd like to save information about someone's [best 90s songs](https://www.youtube.com/playlist?list=PL7DA3D097D6FDBC02), you'd start the program like this:  
+`java -jar YouTubePlaylister.jar https://www.youtube.com/playlist?list=PL7DA3D097D6FDBC02 playlistOutput.json`.
+
+#### 2. IntelliJ IDEA
+First, clone this repository with `git clone https://github.com/ajvazz/YouTube-Playlister.git`. Then, open IntelliJ IDEA and import this program (as Maven). Since the program is run via _program arguments_, you would need to go to Edit Configurations in IntelliJ IDEA and under `Program arguments` just pass YouTube playlist's link and a file output, the same as you would in a console.
 **Note**: there might be some IntelliJ-based problems with initial program start up. Feel free to contact me if you have any questions.
 
 ## Usage
+When you run this program as stated above (either via console or IntelliJ), an output file with playlist information will appear. It is located in path you provided when you started the program.
```

Comment:
playlist output file should be optional. Default value should be provided. At the end of execution program should print the location of the output file

Reply:
playlist ID is mandatory argument. If not provided, program should terminate and print appropriate message to the user.